boa_engine/src/bytecompiler/statement/loop.rs

```diff
@@ -22,7 +22,7 @@ impl ByteCompiler<'_, '_> {
         configurable_globals: bool,
     ) -> JsResult<()> {
         self.context.push_compile_time_environment(false);
-        let push_env = self.emit_and_track_decl_env();
+        let push_env = self.emit_opcode_with_two_operands(Opcode::PushDeclarativeEnvironment);
         self.push_empty_loop_jump_control();
```

Comment:
This was a small change that was added during the `JumpControlInfo` being added. Set label and start address occur down further from lines 46-51. The initial idea was to make everything read a bit more uniform and not insert pushing the jump info 15+ lines into the compilation. But I think it could be taken or left either way.